docs/tutorials/fundamentals/installation.md

```diff
@@ -184,6 +185,10 @@ pip install "napari[pyqt5]"    # for PyQt5
 pip install "napari[pyside2]"  # for PySide2
 ```
 
+*(On arm64 macOS, it is the availability of wheels for these Qt5 backends via pip
+that is the problem. If you provide the backend, for example via conda from conda-forge,
+then you can `pip install napari`, but without the `[all]`.)*
```

Comment:
Instead of adding this after the fact, I would instead update the line above:

> Running `pip install "napari[all]"` will install the default framework — currently PyQt5, but this could change in the future.

to something like:

````markdown
Running `pip install "napari[all]"` will install the default framework —
currently PyQt5, but this could change in the future. (But note: if you
have a Mac machine with Apple Silicon — a processor with a name like
"M1" — see {ref}`note-m1`.)

```{note}
:name: note-m1

For machines with Apple Silicon, pre-compiled PyQt5 or PySide2
[wheels](https://realpython.com/python-wheels/) are not available,
so trying to `pip install napari[all]` will fail. You can install one of
those libraries separately, for example with `conda` or `mamba`,
and then `pip install napari`.
```
````

Reply:
@jni I've updated for your comment. Slightly tweaked in a way that made more sense to me.
What do you think @GenevieveBuckley comment to put conda-forge first?
From my year+ it seems like conda vs pip is like religious wars—I'm not sure where we stand, officially. But we do provide instructions for conda envs, so ... 👀